components/Calendar/AdminCalendar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Calendar from "@fullcalendar/react";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import timeGridPlugin from "@fullcalendar/timegrid";
+
+const AdminCalendar: React.FC<object> = () => {
+  const _ = dayGridPlugin;
```

Comment:
Can we add a comment here saying "using daygrid plugin to allow time grid plugin to load"